source/module_surchem/force.cpp

```diff
@@ -0,0 +1,176 @@
+#include "surchem.h"
+#include "../module_base/timer.h"
+
+void force_cor_one(const UnitCell &cell, ModulePW::PW_Basis* rho_basis , ModuleBase::matrix& forcesol)
+{
+   
+   
+    //delta phi multiply by the derivative of nuclear charge density with respect to the positions
+    std::complex<double> *N = new std::complex<double>[rho_basis->npw];
+    std::complex<double> *vloc_at = new std::complex<double>[rho_basis->npw];
+    std::complex<double> *delta_phi_g = new complex<double>[rho_basis->npw];
+    //ModuleBase::GlobalFunc::ZEROS(delta_phi_g, rho_basis->npw);
+
+    GlobalC::UFFT.ToReciSpace(GlobalC::solvent_model.delta_phi, delta_phi_g,rho_basis);
```

Comment:
https://github.com/deepmodeling/abacus-develop/blob/develop/source/src_pw/H_Hartree_pw.cpp#L29 UFFT is unused and this is the new interface of FFT